ProcessLib/ProcessVariable.cpp

```diff
@@ -89,6 +89,9 @@ ProcessVariable::ProcessVariable(BaseLib::ConfigTree const& config,
             auto const component_id =
                     //! \ogs_file_param{boundary_condition__component}
                     bc_config.getConfigParameter<int>("component", 0);
+            if (_n_components > 1) {
+                bc_config.peekConfigParameter<int>("component");
```

Comment:
I thought peek after get doesn't work. At least I didn't want it to work in that order. ^^